### ISSUE TITLE
Show error while implementation

### ISSUE THREAD

**Issue by bawakali** [hassaanjamil/hj-qibla-compass#3]
Failed to resolve: com.hassanjamil:qibla:0.2.2
Show in Project Structure dialog
Affected Modules: app

_____________________________________

Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.

**Comment by hassaanjamil** [comment]
Yes, I get it. Development was paused on this project, I made changes and publish it early and with fewer/limited features. Anyways, I am planning 1.0 and you will find your problem catered in it. Thanks for patience.

**Comment by hani-28** [comment]
A simple fix to this can be by updating readme instruction to following:
    maven {
        url "http://api.tplmaps.com:8081/artifactory/example-repo-local/"
        allowInsecureProtocol = true
    }

**Comment by hassaanjamil** [comment]
@hani-28 The pain will be treated as I will deploy the library to maven.

> A simple fix to this can be by updating readme instruction to following: maven { url "http://api.tplmaps.com:8081/artifactory/example-repo-local/" allowInsecureProtocol = true }

**Comment by hassaanjamil** [comment]
> A simple fix to this can be by updating readme instruction to following: `maven { url "http://api.tplmaps.com:8081/artifactory/example-repo-local/" allowInsecureProtocol = true }`

@hani-28 The pain will be removed as I deploy the library to maven.

**Comment by hassaanjamil** [comment]
> A simple fix to this can be by updating readme instruction to following: maven { url "http://api.tplmaps.com:8081/artifactory/example-repo-local/" allowInsecureProtocol = true }

@hani-28 The pain will be removed as I deploy the library to maven.
